composer.json

```diff
@@ -23,14 +23,14 @@
         }
     ],
     "require":     {
-        "php": "^5.4|^7",
+        "php": "^5.5.9|^7",
```

Comment:
Might be an idea to copy the exact requirement the php bridge uses instead?

see https://github.com/symfony/phpunit-bridge/blob/4.4/composer.json

Reply:
https://twitter.com/nicolasgrekas/status/1263023258938548225

Reply:
Do you mean just relaxing the requirement, or copying the whole explanation as well?

Reply:
@Daimona I got it as a suggestion to switch to ">=5.5.9" , which is not technically relaxing the requirement (apart from the future PHP 8 compatibility, if this is what you meant)

Reply:
Yeah, sounds pretty clear now, I began writing my reply before the link was posted, and didn't see it.
(Of note, yes, I was indeed hinting to PHP 8 compatibility)